presto-base-arrow-flight/pom.xml

```diff
@@ -288,14 +289,22 @@
             <dependency>
                 <groupId>org.jetbrains.kotlin</groupId>
                 <artifactId>kotlin-stdlib-common</artifactId>
-                <version>1.6.20</version>
+                <version>${kotlin.version}</version>
             </dependency>
 
             <dependency>
                 <groupId>com.google.errorprone</groupId>
                 <artifactId>error_prone_annotations</artifactId>
-                <version>2.23.0</version>
+                <version>${error_prone_annotations}</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.apache.arrow</groupId>
+                <artifactId>arrow-algorithm</artifactId>
+                <version>18.0.0</version>
```

Comment:
can't the arrow version be specified in properties?

Reply:
this version is specific only to arrow-algorithm , Its used only once . should we move this to properties?

Reply:
isn't there already an arrow.version property defined, can't we use that? is this version different?

Reply:
yeah this is 18 , anyways i will change it to 17 and recheck

Reply:
then why not set all arrow dependencies to 18

Reply:
okay will change it accordingly to 18.1.0 which is latest , will re run the test cases